lib/plugins/aws/invoke/index.test.js

```diff
@@ -99,6 +101,14 @@ describe('AwsInvoke', () => {
       });
     });
 
+    it('should parse data if it is a json string', () => {
```

Comment:
We have the same test above ("Should parse JSON data if it is provided via CLI")